### ISSUE TITLE
Support Bugzilla.

### ISSUE THREAD

**Issue by beedell-roke** [super-productivity/super-productivity#2716]
## Problem Statement
I'm unable to access issues that I have created in Bugzilla instances.

## Possible Solution
Support Bugzilla.

## Describe alternatives you've considered
I don't believe that any method except direct API support exists to manage Bugzilla issues.

## Additional context
https://github.com/johannesjo/super-productivity/issues/2717#issue-1848229603 would be rather important due to each userbase hosting a separate instance of the software.

**Comment by RokeJulianLockhart** [comment]
I've filed https://bugzilla.mozilla.org/show_bug.cgi?id=1864874#c0 to hopefully remove the necessity of implementing this.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/johannesjo/super-productivity/issues/2716#issuecomment-1998767317

@GitHub-Actions, it's relevant.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/johannesjo/super-productivity/issues/2716#issuecomment-2472172064

@GitHub-Actions, it remains relevant.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/johannesjo/super-productivity/issues/2716#issuecomment-2723137134

@GitHub-Actions, obviously, it's relevant. Else, I would have closed it. I fail to comprehend the purpose of this automation – its existence appears to cause *more* work, rather than less.

**Comment by beedell-roke** [comment]
> https://github.com/johannesjo/super-productivity/issues/2716#issuecomment-3300920977

It remains relevant. If I ever locate who wrote this bot, I'll slaughter them.

**Comment by johannesjo** [comment]
PRs are welcome! I probably won't work on this myself.

**Comment by OptionalLion411** [comment]
@RokeJulianLockhart, could you explain what exact features you expect from a custom issue provider integration for Bugzilla? What data is missing?
I think using the iCal provider integration using the iCal option mentioned in https://bugzilla.mozilla.org/show_bug.cgi?id=1864874#c0 should be fine...

**Comment by beedell-roke** [comment]
> I think using the iCal provider integration using the iCal option mentioned in https://bugzilla.mozilla.org/show_bug.cgi?id=1864874#c0 should be fine...

@OptionalLion411, that would be a solution for Mozilla's Bugzilla instance at least (which is why I filed that bug). However, I've no guarantee that Mozilla would implement it, that it would be upstreamed to Harmony, that that most instances would update it, whereas their API is stable across potentially downstream-modified instances.

> could you explain what exact features you expect from a custom issue provider integration for Bugzilla? What data is missing?

I expect an SP integration to act as the aforeferenced BZ ticket describes: it would enumerate any user-filed and assigned tickets as tasks, and whether they're `UNCONFIRMED`/`NEW`/`ASSIGNED`/`REOPENED` versus `RESOLVED` would correspond to their `STATUS:CONFIRMED`, versus `CANCELLED` or [a `VTODO` with] `COMPLETE`.
